web/src/components/Breadcrumbs.tsx

```diff
@@ -1,201 +1,90 @@
   }
 
-  & li {
+  .${breadcrumbsClasses.li} {
     overflow: hidden;
 
     &:first-of-type {
       overflow: visible;
     }
 
-    /* Shrink last breadcrumb item more if it has more than 2 items on small screens to make previous items more readable */
-    ${props => props.theme.breakpoints.down('sm')} {
+    ${props => props.theme.breakpoints.between('sm', 'md')} {
       &:last-of-type {
-        max-width: ${props => (props.length > 2 ? '100px' : 'none')};
+        max-width: 50%;
       }
     }
   }
 
-  & li:nth-of-type(even) {
-    overflow: visible;
-    flex-shrink: 0;
-    min-width: 16px;
-  }
-
-  /* Collapsed MuiBreadcrumbs */
-  /* stylelint-disable-next-line selector-class-pattern */
-  & .MuiButtonBase-root {
-    justify-self: center;
-    margin-top: 5px;
-  }
-
-  & li:has([data-ellipsis]) {
-    overflow: visible;
-  }
-
-  /* stylelint-disable-next-line selector-class-pattern */
-  & li:has(.MuiIconButton-root) {
+  & li:has(.${iconButtonClasses.root}) {
     overflow: visible;
-    margin-top: 0;
   }
 `
 
-const StyledIcon = styled(Icon)`
-  width: 24px;
-  height: 24px;
-`
-
-const StyledLink = styled(Link)`
-  margin-inline-end: 4px;
-  flex-shrink: 0;
-  overflow: visible;
-`
-
-const Separator = styled('div')`
-  &::before {
-    color: ${props => props.theme.legacy.colors.textColor};
-    font-size: 19px;
-    content: ' > ';
-  }
-`
-
-const StyledIconButton = styled(IconButton)`
-  white-space: nowrap;
-  flex-shrink: 0;
-  min-width: auto;
-  margin-top: 0 !important;
-  color: ${props => props.theme.palette.text.secondary};
-`
-
 type BreadcrumbsProps = {
-  ancestorBreadcrumbs: BreadcrumbModel[]
-  currentBreadcrumb: BreadcrumbModel
+  breadcrumbs: BreadcrumbModel[]
 }
 
-const getBreadcrumbs = (
-  ancestorBreadcrumbs: BreadcrumbModel[],
-  currentBreadcrumb: BreadcrumbModel,
-): { breadcrumbs: BreadcrumbModel[]; collapsedBreadcrumbs: BreadcrumbModel[] } => {
-  const allBreadcrumbs = [...ancestorBreadcrumbs, currentBreadcrumb]
-  const breadCrumbsLimit = 3 // with home included
-  const lastTwoCrumbs = -2
-
-  if (ancestorBreadcrumbs.length === 0) {
-    return { breadcrumbs: [], collapsedBreadcrumbs: [] }
-  }
-
-  if (allBreadcrumbs.length <= breadCrumbsLimit) {
-    return { breadcrumbs: allBreadcrumbs, collapsedBreadcrumbs: [] }
-  }
+const Breadcrumbs = ({ breadcrumbs }: BreadcrumbsProps): ReactElement | null => {
+  const [menuAnchorElement, setMenuAnchorElement] = useState<HTMLButtonElement | null>(null)
+  const { small } = useDimensions()
+  const { t } = useTranslation('common')
 
-  const home = allBreadcrumbs[0] as BreadcrumbModel
-  const rest = allBreadcrumbs.slice(1)
+  const openMenu = (event: React.MouseEvent<HTMLButtonElement>) => setMenuAnchorElement(event.currentTarget)
+  const closeMenu = () => setMenuAnchorElement(null)
 
-  const ellipsis = new BreadcrumbModel({
-    title: '...',
-    pathname: '',
-    node: null,
-  })
+  const maxVisible = small ? 1 : 2
```

Comment:
What do you think about this? Should we only display one item on xsmall viewports (<600px)? Or do you think we should keep the current logic?

When merging main this has to be changed to:
```suggestion
  const maxVisible = xsmall ? 1 : 2
```

Reply:
From what we gathered from Toni we can do one or two so keep it with one item as you did (less ellipsis used).

<img width="742" height="90" alt="image" src="https://github.com/user-attachments/assets/64e6e9a4-0003-4fe1-871b-6cdf71d7bd96" />